castervoice/rules/ccr/rust_rules/rust2.py

```diff
@@ -13,12 +13,16 @@ class RustNon(MappingRule):
             R(Text("panic!()") + Key("left")),
         "macro assertion":
             R(Text("assert_eq!()") + Key("left")),
+        "macro debug":
+            R(Text("dbg!(&)") + Key("left")),
         "ternary":
             R(Text("if TOKEN == TOKEN { TOKEN } else { TOKEN }")),
         "function [<return>]":
             R(Text("fn TOKEN(TOKEN)%(return)s{}")),
         "infinite loop":
             R(Text("loop {}") + Key("left")),
+        "unwrap":
+            R(Text(".unwrap()"))
```

Comment:
Though it is kind of frowned upon in the rust documentation, unwrapping options and results is a common thing in rust, so I added this here.